### ISSUE TITLE
Refine route

### ISSUE THREAD

**Issue by samuliheljo** [HSLdevcom/digitransit-ui#106]
Refine routing as defined in concept here:
https://projects.invisionapp.com/share/MY2F0CQ2W#/screens/68574795

![refine](https://cloud.githubusercontent.com/assets/1298522/9541604/7c36ec28-4d72-11e5-814c-bb5e16fe502a.png)

Basic logic:
- [x] When route summaries page loads, we will preselect all transport modes from first row as well as "walking man" on second button row.
- [ ] Modifications are applied when offcanvas closes. This means that we do new routing query only once when selections are made.

Tasks:
- [x] functionality for all transport modes. (top row)
- [x] functionality for walking penalty
- [x] functionality for transfers
- [x] functionality for transfer marginal
- [x] functionality for walking speed
- [x] functionality for wheelchair accessibility

Out of scope for this task
- Ticketing zone
- Via point
- Filter lines
- Bicycling
- Kiss and ride

**Comment by MikaelStenstrand** [comment]
as discussed, the changes are reflected directly on the given route and not when the offcanvas is closed
